specs/_features/maxeb_increase/capella.py

```diff
@@ -1627,14 +1646,37 @@ def process_pending_balance_deposits(state: BeaconState) -> None:
     state.pending_balance_deposits = state.pending_balance_deposits[next_pending_deposit_index:]
 
 
+def apply_pending_consolidation(state: BeaconState, pending_consolidation: PendingConsolidation) -> None:
+    source_validator = state.validators[pending_consolidation.source_index]
+    target_validator = state.validators[pending_consolidation.target_index]
+    # Move active balance
+    active_balance_ceil = MIN_ACTIVATION_BALANCE if has_eth1_withdrawal_credential(source_validator) else MAX_EFFECTIVE_BALANCE
```

Comment:
does this work with custom ceilings ?

Reply:
Yes, it should be replaced with the ceiling computation function, I guess if we have custom ceilings it won’t be the only place where such function is needed.